contracts/governance/InterchainGovernance.sol

```diff
@@ -174,7 +174,8 @@ contract InterchainGovernance is AxelarGMPExecutable, TimeLock, Caller, IInterch
     }
 
     /**
-     * @dev Get proposal hash using the target, callData, and nativeValue
+     * @notice Get proposal hash using the target, callData, and nativeValue
+     * @return The proposal hash
```

Comment:
This is another contract fixed in production. Prefer to avoid changing it unless needed

Reply:
Got it. Reverted change.